doc/flux_to_redux.md

```diff
@@ -0,0 +1,257 @@
+# Flux to Redux Transition Guide
+> A document explaining the fundamentals of transitioning a Flux architecture to a Redux architecture.
+>
+> NOTE: Before attempting to transition, make sure you understand the content from the [OverReact Redux Documentation](./over_react_redux_documentation.md). This guide covers differences between Flux to Redux but does not try to comprehensively describe Redux.
+---
+* __[Goal](#goal)__
+* __[Do I have to transition?](#do-i-have-to-transition-to-redux)__
+* __[A Simple Example](#a-simple-example)__
+    * [A Basic Flux App](#a-basic-flux-app)
+    * [The Redux Version](#the-redux-version)
+    * [Basic Example: The Main Differences](#basic-example:-the-main-differences)
+    * [Basic Conversion Step by Step](#basic-conversion-step-by-step)
+* __[An Advanced Example](#an-advanced-example)__
+    * [An Advanced Flux App](#an-advanced-flux-app)
+    * [Converted to Redux](#converted-to-redux)
+    * [Advanced Conversion Step by Step](#advanced-conversion-step-by-step)
+    * [What if the Advanced Conversion is too Daunting?](#what-if-the-advanced-conversion-is-too-daunting?)
+* __[Influx Architecture](#influx-architecture)__
```

Comment:
I definitely see your point, and I could go either way. Do you feel strongly? I lean towards keeping Influx because I feel like it is actually pretty descriptive to the situation of the code and isn't a lot to comprehend for the few that will need to worry about it.